otter/models/cass.py

```diff
@@ -738,6 +739,22 @@ def _modify_state():
                               log.bind(category='locking'), acquire_timeout=150,
                               release_timeout=30)
 
+    def update_status(self, status):
+        """
+        see :meth:`otter.models.interface.IScalingGroup.update_status`
+        """
+        self.log.bind(status=status).msg("Updating status")
+
+        @self.with_timestamp
+        def _do_update(ts, _):
+            return self.connection.execute(
+                _cql_update.format(cf=self.group_table, column='status', name=':status'),
+                {'tenantId': self.tenant_id, 'groupId': self.uuid, 'ts': ts,
+                 'status': status.name},
+                self.get_consistency('update', 'status'))
```

Comment:
Really sorry about this, since this PR preceded the other one with quorum.  :|  But this is probably no longer applicable.